WORKSPACE

```diff
@@ -26,10 +26,11 @@ versions.check(
 
 http_archive(
     name = "io_bazel_rules_webtesting",
-    sha256 = "9bb461d5ef08e850025480bab185fd269242d4e533bca75bfb748001ceb343c3",
+    sha256 = "41d500a97ad9621dcf92fcb0cd77916e517388b196e5c3f0e63c7753e983b2bb",
+    strip_prefix = "rules_webtesting-4d7ec75d1cbb289f977b41638fc8b630bdf22bee",
     urls = [
-        "http://mirror.tensorflow.org/github.com/bazelbuild/rules_webtesting/releases/download/0.3.3/rules_webtesting.tar.gz",
-        "https://github.com/bazelbuild/rules_webtesting/releases/download/0.3.3/rules_webtesting.tar.gz",
+        "http://mirror.tensorflow.org/github.com/bazelbuild/rules_webtesting/archive/4d7ec75d1cbb289f977b41638fc8b630bdf22bee.tar.gz",
```

Comment:
I think all of the issues we encountered with `platform_http_file` and `web_test_archive` rules being removed are due to this change.

But I don't understand why we're changing this reference... I thought we only had to pin the one below, related to "closure" rules.

And I don't understand what version/snapshot we are referencing here. (Maybe we should add a comment?)

I suppose it's fine to update it, and we'll need to make these changes whenever we do, anyway... But I think we could make these changes independently. Or is there a reason to make them together?